api/user.go

```diff
@@ -13,6 +13,15 @@ import (
 	"github.com/miniflux/miniflux/http/response/json"
 )
 
+
+// Current is the API handler to retrieve the user id
+func (c *Controller) Current(w http.ResponseWriter, r *http.Request) {
+	type UserMeta struct {
+		UserID int64 `json:"user_id,omitempty"`
+	}
+	json.OK(w, UserMeta { UserID: context.New(r).UserID() })
```

Comment:
What about returning the whole user object? it might be more useful than just the user ID.

You can do something like that:

```go
user, err := c.store.UserByID(userID)
if err != nil {
	json.ServerError(w, err)
	return
}

json.OK(w, user)
```